### PR TITLE
Increase consolidation inputs

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Increase max consolidation inputs amount

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Modified example

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/account/constants.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@ pub(crate) const DEFAULT_OUTPUT_CONSOLIDATION_THRESHOLD: usize = 100;
 #[cfg(feature = "ledger_nano")]
```
Comment (thibault-martinez) [inline src/account/constants.rs]: Put the attribute after the comment

File: src/account/operations/output_consolidation.rs [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+    },
 };
 #[cfg(feature = "ledger_nano")]
 use iota_client::secret::SecretManager;
 
+// Constants for the calculation of the amount of inputs we can use with a ledger nano
+#[cfg(feature = "ledger_nano")]
+const ESSENCE_SIZE_WITHOUT_IN_AND_OUTPUTS: usize = 49;
+#[cfg(feature = "ledger_nano")]
+const INPUT_SIZE_IN_ESSENCE: usize = 43;
```
Comment (thibault-martinez) [inline src/account/operations/output_consolidation.rs]: How do you get 43 ?

File: src/account/operations/output_consolidation.rs [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+#[cfg(feature = "ledger_nano")]
+const MIN_OUTPUT_SIZE_IN_ESSENCE: usize = 46;
```
Comment (thibault-martinez) [inline src/account/operations/output_consolidation.rs:21]: Same question, I get 45 but I'm probably missing something

File: src/account/operations/output_consolidation.rs (hunk 2 above)
Comment (Thoralf-M) [reply src/account/operations/output_consolidation.rs:18]: Got the number from Thomas
```Rust
    let input = Input::Utxo(UtxoInput::from_str("0xb66fd384cb5755668f1890ea2e41d699db9cf32f3bc422ad3c24ffeb9c7f01d00000").unwrap());
    println!("{}", input.packed_len());
```
prints 35, so I guess this is the correct number to use?

Comment (Thoralf-M) [reply src/account/operations/output_consolidation.rs]: Got the number and din't think further about it :trollface: 
It's 35 + 8 for the bip32 path part we need in addition to it, will add a comment and change the name

File: src/account/operations/output_consolidation.rs (hunk 3 above)
Comment (Thoralf-M) [reply src/account/operations/output_consolidation.rs:21]: I get 45 for a basic output, but then we need to also add the output type byte

File: src/account/operations/output_consolidation.rs (hunk 2 above)
Comment (thibault-martinez) [reply src/account/operations/output_consolidation.rs]: Ah I see, wasn't accounting for the bip32 path, thanks

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**